### PR TITLE
Use and expose the .env file for the build.

### PR DESCRIPTION
**Pull request by FabienLelaquais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,13 +191,23 @@ the environment variable "TAIPY_GUI_DIR" to the value indicated at the "Location
 You can verify that this setting is correct by confirming that there is a directory at the
 location `$TAIPY_GUI_DIR/taipy/gui/webapp`.
 
-Once the environment variable "TAIPY_GUI_DIR" is set, here are the steps to setup the build:
+You can store this setting in the file `taipy_gui_ext_library/front-end/.env` if you want
+to run the build process several times and not forget to set the environment variable
+each time.<br/>
+This file must contain a single line:
```
Comment (FredLL-Avaiga) [inline README.md]: Must contain is enough
They can put whatever they like in this file

File: README.md [hunk 2]
```diff
@@ -191,13 +191,23 @@ the environment variable "TAIPY_GUI_DIR" to the value indicated at the "Location
...
+
+Once the environment variable "TAIPY_GUI_DIR" is set (as an environment variable or in the
+`.env` file next to `package.json`), here are the steps to setup the build:
 
 - Change your directory to where the front-end code is located:<br/>
   `cd taipy_gui_ext_library/front-end`
 - Install the packages that your library depends on:<br/>
   `npm install`<br/>
-  This will run a JavaScript script that installs the Taipy GUI Extension API library.
+  This will run a JavaScript script that installs the Taipy GUI Extension API library.<br/>
```
Comment (FredLL-Avaiga) [inline README.md]: A NodeJs script?

File: README.md [hunk 3]
```diff
@@ -191,13 +191,23 @@ the environment variable "TAIPY_GUI_DIR" to the value indicated at the "Location
...
+  This command will fail if the environment variable "TAIPY_GUI_DIR" is not set properly.
 
 The 'front-end' directory will have an additional subdirectory called 'node_modules' where
 all dependent libraries are copied.
```
Comment (FredLL-Avaiga) [inline README.md]: Downloaded?

File: taipy_gui_ext_library/front-end/webpack.config.js [hunk 4]
```diff
@@ -11,16 +11,13 @@
...
-// webpack should be in the node_modules directory, install if not.
-const path = require("path");
 const webpack = require("webpack");
+const path = require("path");
 require("dotenv").config();
 
-module.exports = (env, options) => {
-  const TAIPY_GUI_DIR = process.env.TAIPY_GUI_DIR;
-
+module.exports = (_env, options) => {
```
Comment (FredLL-Avaiga) [inline taipy_gui_ext_library/front-end/webpack.config.js:18]: Why _env?

**Review by FredLL-Avaiga — APPROVED** [review]
🧠

File: README.md (hunk 2 above)
Comment (FabienLelaquais) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (FabienLelaquais) [reply README.md]: Installed?

File: taipy_gui_ext_library/front-end/webpack.config.js (hunk 4 above)
Comment (FabienLelaquais) [reply taipy_gui_ext_library/front-end/webpack.config.js:18]: Because it is not used, and I have a warning in my IDE. No pain.

**Pull request merged**